src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.SupportedTargetPlatforms.props

```diff
@@ -0,0 +1,21 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.SupportedTargetPlatforms.props
```

Comment:
Since we renamed the `SupportedTargetPlatform` item to `SupportedPlatform`, we probably want to rename this file as well.